src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -2618,9 +2618,10 @@ private StatementAst SwitchStatementRule(LabelToken labelToken, Token switchToke
                         specifiedFlags.Add("parallel", new Tuple<Token, Ast>(switchParameterToken, null));
                     }
                 }
-                else if (IsSpecificParameter(switchParameterToken, "file"))
+                else if (IsSpecificParameter(switchParameterToken, "file") || IsSpecificParameter(switchParameterToken, "literalfile"))
                 {
-                    flags |= SwitchFlags.File;
+                    bool isLiteralPath = IsSpecificParameter(switchParameterToken, "literalfile");
```

Comment:
Is there a way to avoid calling this method twice? 

Or is calling it sufficiently cheap that it doesn't really matter?

Reply:
It is simple method. I don't performance problem here.

And we use "file" and "literalpath" about 6 times. Using const will be good.

Reply:
I imagine the ideal solution is to turn the body of the if statement into a method and have two `else if`s:

```csharp
else if (IsSpecificParameter(switchParameterToken, "file")
{
     WhateverTheBodyDoes(..., flags | SwitchFlags.File);
}
else if (IsSpecificParameter(switchParameterToken, "literalPath")
{
    DoWhateverTheBodyDoes(..., flags | SwitchFlags.LiteralFile);
}
```